targets/teensy40.json

```diff
@@ -0,0 +1,25 @@
+{
+  "inherits": ["cortex-m"],
+  "llvm-target": "armv7em-none-eabi",
+  "cpu": "cortex-m7",
+  "build-tags": ["teensy40", "teensy", "mimxrt1062", "nxp"],
+  "automatic-stack-size": false,
+  "default-stack-size": 131072,
+  "cflags": [
+    "--target=armv7em-none-eabi",
+    "-Qunused-arguments",
+    "-mfloat-abi=hard",
+    "-mfpu=fpv5-d16"
```

Comment:
Unfortunately, floating point is not yet supported in TinyGo stack switching so I would suggest using softfloat until it is. Otherwise, if multiple goroutines are using the FPU there is a real risk of one goroutine corrupting the float registers of the other goroutine.

That said, these flags only apply to C code so are of little importance to most Go code (and in fact would introduce a binary incompatibility between Go and C).

Reply:
Interesting! Will remove, thanks.